superbench/tools/system_info.py

```diff
@@ -76,14 +81,14 @@ def parse_key_value_lines(self, lines, required_keywords=None, omitted_values=No
                 i += 1
                 continue
             # process with indent recursively
-            indent = self.__count_prefix_indent(lines[i])
-            if i + 1 < length and self.__count_prefix_indent(lines[i + 1]) > indent:
+            indent = self.count_prefix_indent(lines[i])
+            if i + 1 < length and self.count_prefix_indent(lines[i + 1]) > indent:
```

Comment:
why not change `def count_prefix_indent()` to `def __count_prefix_indent()`?

Reply:
changed